website/docs/configuration/provider-requirements.html.md

```diff
@@ -0,0 +1,299 @@
+---
+layout: "docs"
+page_title: "Provider Requirements - Configuration Language"
+---
+
+## Provider Requirements
+
+-> **Note:** If you are using Terraform 0.11 or
+earlier, see
+[0.11 Configuration Language: Provider Versions](../configuration-0-11/providers.html#provider-versions) instead.
+
+Terraform relies on plugins called "providers" to interact with remote systems.
+Each provider offers a set of named
+[resource types](resources.html#resource-types-and-arguments), and defines for
+each resource type which arguments it accepts, which attributes it exports, and
+how changes to resources of that type are actually applied to remote APIs.
+
+You can discover publically-available providers
```

Comment:
```suggestion
You can discover publicly-available providers
```

(I do that one a lot!)

Reply:
"Providers that are available in the public alley."